### PR TITLE
Added mandatory location hint messages

### PR DESCRIPTION
**Pull request by MattEqualsCoder**

### CODE REVIEW

File: Profiles/Sassy/responses.yml [hunk 1]
```diff
@@ -842,6 +846,9 @@ Chat:
   ModeratorClosedGuessingGameBeforeStarting:
   - Text: Pay attention, {0}. We haven't even started yet
   - Text: Close what, {0}? The guessing game never started.
+  - Text: I know, {0}. I'm ready for this to be over too. But we haven't started the guessing game yet.
+  - Text: I think {0} is telling you to go faster as they wanted to start the guessing game
+  - Text: Are you even watching, {0}? We have started the guessing game yet.
```
Comment (Vivelin) [inline Profiles/Sassy/responses.yml:851]: I assume you meant "We have<ins>n't</ins> started"?

Comment (MattEqualsCoder) [reply Profiles/Sassy/responses.yml]: Maybe one day I'll think to proofread my stuff before creating PRs!

File: Profiles/BCU/responses.yml [hunk 2]
```diff
@@ -101,6 +101,11 @@ Chat:
     madsteve79: Mad Steve 79
     20thCenturyDinosaur: 20th Century Dinosaur
     Kefka40: Kefka 40
+    PyroHeromatt: Pyro Hero Matt
+    DrMutagen: Doctor Mutagen
+    Nightbot: Night bot
```
Comment (CPColin) [inline Profiles/BCU/responses.yml:106]: I'm gonna miss "Night Butt"

**Review by CPColin — APPROVED** [review]

File: Profiles/BCU/responses.yml (hunk 2 above)
Comment (MattEqualsCoder) [reply Profiles/BCU/responses.yml:106]: We need to allow these to be randomized or something. lol

**Pull request merged**